samples/app/package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "gandi.translate-sample",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "start": "react-scripts start",
+    "build": "react-scripts build",
+    "test": "react-scripts test --env=jsdom",
+    "eject": "react-scripts eject",
+    "i18n": "react-translate-scripts"
+  },
+  "devDependencies": {
+    "babel-cli": "^6.18.0",
+    "babel-gettext-plugin": "^2.2.0",
```

Comment:
my bad, I forgot to re-add this in normal deps of react-translate: https://github.com/Gandi/react-translate/pull/4#issuecomment-262727340

Do you mind fix this in this PR, or I provide new one ?